src/redux/reducers/lookupApi.test.js

```diff
@@ -117,3 +117,23 @@ test("the authenticated user's profile is set in self", () => {
   // check the state wasn't mutated
   expect(Object.is(initialState.self, nextState.self)).toBe(false);
 });
+
+test('institutions are copied from institution list success action', () => {
+  const initialState = reducer(undefined, { type: 'not-an-action' });
+  expect(initialState.institutions.fetchedAt).toBeNull();
+  expect(initialState.institutions.byInstid.get('AAA')).toBeUndefined();
+
+  const action = {
+    type: INSTITUTIONS_LIST_SUCCESS,
+    payload: { results: [
+      { instid: 'AAA', name: 'Dept of A' },
+      { instid: 'BBB', name: 'Dept of B' },
+    ] },
+  };
+
+  const nextState = reducer(initialState, action);
+
+  expect(nextState.institutions.fetchedAt).not.toBeNull();
+  expect(nextState.institutions.byInstid.get('AAA')).toEqual({ instid: 'AAA', name: 'Dept of A' });
+  expect(nextState.institutions.byInstid.get('BBB')).toEqual({ instid: 'BBB', name: 'Dept of B' });
+});
```

Comment:
you could add a test to check for immutability

Reply:
Done